planning/behavior_velocity_planner/include/scene_module/occlusion_spot/occlusion_spot_utils.hpp

```diff
@@ -71,24 +71,22 @@ struct Sidewalk
   double slice_size;               // [m] size of each slice
   double min_occlusion_spot_size;  // [m] minumum size to care about the occlusion spot
 };
-
-struct VehicleInfo
+struct Velocity
 {
-  double vehicle_width;      // [m]  vehicle_width from parameter server
-  double baselink_to_front;  // [m]  wheel_base + front_overhang
-};
-
-struct EgoVelocity
-{
-  double ebs_decel;     // [m/s^2] emergency braking system deceleration
-  double pbs_decel;     // [m/s^2] predictive braking system deceleration
-  double min_velocity;  // [m/s]   minimum allowed velocity not to stop
+  double safety_ratio;  // [-] safety margin for planning error
+  double j_max;         // [m/s^3] emergency braking system jerk
+  double a_max;         // [m/s^2] emergency braking system deceleration
+  double a_min;         // [m/s^2] maximum allowed deceleration
```

Comment:
j/a_max is minimum value. So, should it be j/a_min ?

Reply:
@tkimura4 
a_min is  a deceleration from current ego velocity so I changed it to `max_slow_down_accel` which could be more accurate